src/mscorlib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -777,6 +783,42 @@ public int EnsureCapacity(int capacity)
             return newSize;
         }
 
+        /// <summary>
```

Comment:
you'll want to make the comments on the two overloads consistent with each other